artifactory/commands/golang/archive.go

```diff
@@ -134,8 +119,13 @@ func archiveProject(writer io.Writer, dir, mod, version string) error {
 				return filepath.SkipDir
 			}
 		}
+
 		if info.Mode().IsRegular() {
-			if !isVendoredPackage(slashPath) {
+			excluded, err := isPathExcluded(filePath, excludePathPattern)
```

Comment:
This code will not work properly, if `excludePathPattern` includes multiple patterns. The name of the `excludePathPattern` is wrong, and hints that this string can include only one pattern. 
The test you added probably does not test a case where multiple patterns are sent.